Formula/golo.rb

```diff
@@ -1,20 +1,17 @@
 class Golo < Formula
   desc "Lightweight dynamic language for the JVM"
   homepage "https://golo-lang.org/"
-  url "https://github.com/eclipse/golo-lang/releases/download/release%2F3.4.0/golo-3.4.0.zip"
+  url "https://github.com/eclipse-archived/golo-lang/releases/download/release%2F3.4.0/golo-3.4.0.zip"
   sha256 "867c462a41a20e4b7dc1aef461b809d193a505c2a757477b147f0e30235bd545"
   license "EPL-2.0"
-  head "https://github.com/eclipse/golo-lang.git", branch: "master"
-
-  livecheck do
-    url :stable
-    regex(%r{^release/v?(\d+(?:\.\d+)+)$}i)
-  end
+  head "https://github.com/eclipse-archived/golo-lang.git", branch: "master"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "069c041fccb8a95fd9ea3c7a9c2105b384433a0e724d5b48812aa37d0d5c9f2d"
   end
 
+  deprecate! date: "2023-01-01", because: :deprecated_upstream
```

Comment:
I think it should be `repo_archived`?

Reply:
I feel the wording is kind of nuance when I did the #119534

Reply:
There's some ambiguity around when to use which reason (I'm not sure if we have any clear guidance on this) but I don't have any problem with `:repo_archived` here.

In general, I've tended to use `:repo_archived` for formulae where the `homepage` is simply the repository and `:deprecated_upstream` when the project has a separate website where they acknowledge that the project is end of life, etc.

Reply:
yeah, I feel like `:repo_archived` is a stronger wording than `:deprecated_upstream`. But that is my personal opinion. :)